components/patient-data-sharing/push-ui/src/main/resources/PhenoTips/PushPatient.xml

```diff
@@ -1715,7 +1715,7 @@ $xwiki.ssx.use('PhenoTips.PushPatient')##
 #foreach ($mapEntry in $configuredServers.entrySet())
   ## make sure PhenomeCentral is enabled/configured, and if it is list it as the first push target
   #if ($mapEntry.key.getServerID() == "PhenomeCentral")
-    (% id="push-patient-pc" name="PhenomeCentral" %)((([[$services.localization.render('phenotips.PushPatient.pushPhenomeCentral')&gt;&gt;path:#]])))
```

Comment:
This is used in the javascript code. HTML5 allows `data-*` attributes which could be used instead, but I'm not sure what kind of parser you're using and whether it would accept such an attribute or not. The other standard attribute that could be used is `title`.

Also note that this code is repeated a few lines below, for all other targets except PhenomeCentral.